requirements-dev.txt

```diff
@@ -1,11 +1,4 @@
 Babel
-
-# Dependency of flake8
-# https://github.com/jaraco/configparser/issues/27
-# https://github.com/jaraco/configparser/issues/30
-configparser!=3.5.2,!=3.5.3,!=3.7.0
 docker>=3.7.0,<3.8.0
-flake8>=3.5.0,<3.6.0
 isort>=4.3.4,<4.4.0
-pycodestyle>=2.3.1,<2.4.0
-sentry-flake8>=0.1.0,<0.2.0
+sentry-flake8==0.1.1
```

Comment:
Does this need to be synced in getsentry too? This will likely override the 0.1.0 pin they have there.

Reply:
Yes, getsentry could use a flake8 upgrade, but that isn't blocking in any way (if that's what you were implying).

Reply:
https://github.com/getsentry/getsentry/pull/3199